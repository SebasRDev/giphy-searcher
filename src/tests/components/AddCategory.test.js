import React from 'react';
import AddCategory from '../../components/AddCategory';
import { shallow } from 'enzyme'

describe('tests for AddCategory', ()=>{

  const setCategories = () => {}
  const wrapper = shallow(<AddCategory setCategories={setCategories}/>);

  test('should match with snapshot', ()=>{
    expect(wrapper).toMatchSnapshot();
  })

})