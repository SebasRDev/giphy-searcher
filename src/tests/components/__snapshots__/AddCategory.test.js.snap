// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for AddCategory should match with snapshot 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
